app/src/main/python/gogdl/dl/objects/v1.py

```
@@ -0,0 +1,168 @@
+        for file in new_manifest.files:
+            new_files.update({file.path.lower(): file})
+        
+        old_files = dict()
+        for file in old_manifest.files:
+            old_files.update({file.path.lower(): file})
+
+        for old_file in old_files.values():
+            if not new_files.get(old_file.path.lower()):
+                comparison.deleted.append(old_file)
+        
+        if type(old_manifest) == v2.Manifest:
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)